ironfish/src/utils/time.ts

```diff
@@ -58,6 +68,29 @@ const renderSpan = (
 
   const parts = []
   let magnitude = 0
+  // Do we want to include years? Is it ok to estimate at 365 days?
+  if (time >= MS_PER_YEAR && (magnitude <= 8 || options?.forceYear)) {
+    const years = Math.floor(time / MS_PER_YEAR)
+    time -= years * MS_PER_YEAR
+    parts.push(`${years.toFixed(0)}y`)
+    magnitude = Math.max(magnitude, 7)
+  }
+
+  // Do we want to include months? Is it ok to estimate at 30 days?
```

Comment:
Yes, I think it's ok to include months and to estimate at 30 days a month.

Reply:
Kept month calculation at 30 days and updated month abbreviation to 'M'. capital M is commonly used in other libraries (e.g. y M d h m s ms)